crates/wash-cli/src/run.rs

```diff
@@ -0,0 +1,266 @@
+use anyhow::{anyhow, Context, Result};
+use clap::Parser;
+use std::path::PathBuf;
+use std::str::FromStr;
+use tracing::warn;
+use wash_lib::cli::{CommandOutput, OutputKind};
+use wash_lib::run::{CtxBuilder, DirPerms, FilePerms as WasmtimeFilePerms, LocalRuntime};
+
+#[derive(Parser, Debug, Clone)]
+pub struct RunCommand {
+    /// Reference to a component. Can be a local file or a registry reference.
+    #[clap(name = "ref")]
+    pub reference: String,
+
+    /// Args to pass to the component.
+    #[clap(name = "args")]
+    pub args: Vec<String>,
+
+    #[clap(flatten)]
+    pub env: EnvironmentOptions,
+
+    #[clap(flatten)]
+    pub dir: DirOptions,
+
+    #[clap(flatten)]
+    pub net: NetworkOptions,
+}
+
+#[derive(Parser, Debug, Clone)]
+pub struct EnvironmentOptions {
+    /// Allow access to specific environment variables.
+    #[clap(long = "allow-env")]
+    pub allow: Vec<String>,
+
+    #[clap(long = "allow-env-prefix")]
+    /// Allow access to any environment variable starting with this prefix.
+    pub allow_prefix: Vec<String>,
+
+    /// Allow access to all environment variable.
+    #[clap(long = "allow-env-all")]
+    pub allow_all: bool,
+}
+
+#[derive(Parser, Debug, Clone)]
+pub struct DirOptions {
+    /// Allow mutable access to a directory tree.
+    ///
+    /// Mutability means files can be deleted and created.
+    /// The format of this argument is:
+    ///
+    /// ```shell
```

Comment:
Those docs are not really pretty when rendered in the help message of the CLI 😢 

If anyone has idea to make those more user-friendly 🙏

Reply:
In this case, it might be best to take the more verbose docs and publish them on https://wasmcloud.com under the command line reference for more information, leaving this to simply be (for example):
```
    /// Allow mutable access to a directory tree. https://wasmcloud.com/docs/cli/run#mutable
    pub mutable: Vec<String>,
```